spec/type.dd

```diff
@@ -24,23 +24,20 @@ $(GNAME TypeCtor):
     $(D inout)
     $(D shared)
 
+$(GNAME PrimaryType):
+    $(GLINK BasicType)
+    $(D __vector) $(D $(LPAREN)) $(GLINK Type) $(D $(RPAREN))
+    $(GLINK TypeCtor)$(OPT) $(D $(LPAREN)) $(GLINK Type) $(D $(RPAREN))
```

Comment:
Should this be without `$(OPT)` for now? This change is from your DIP, but I thought this pull request is only a preparation.